tests/test_cli.sh

```diff
@@ -13,6 +13,11 @@ SHARE_DIR=`mktemp -d '/tmp/composer-share.XXXXX'`
 cp -R ./share/* $SHARE_DIR
 chmod a+rx -R $SHARE_DIR
 
+# explicitly enable sshd for live-iso b/c it is disabled by default
+# due to security concerns (no root password required)
+sed -i 's/--disabled="network,sshd"/--disabled="network"/' $SHARE_DIR/composer/live-iso.ks
```

Comment:
It might be less fragile to just replace the ^services line with a new one instead of editing it like this. Just in case it gets changed in the future.

Reply:
Updated.